src/js/containers/search/filters/KeywordContainer.jsx

```diff
@@ -0,0 +1,70 @@
+/**
+  * KeywordContainer.jsx
+  * Created by Emily Gullo 11/30/16
+  **/
+
+import React from 'react';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+
+import * as searchFilterActions from 'redux/actions/search/searchFilterActions';
+
+import Keyword from 'components/search/filters/keyword/Keyword';
+
+const propTypes = {
+    updateTextSearchInput: React.PropTypes.func,
+    clearFilterType: React.PropTypes.func,
+    keyword: React.PropTypes.string
+};
+
+export class KeywordContainer extends React.Component {
+
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            newKeyword: '',
+            keyword: ''
+        };
+
+        this.submitText = this.submitText.bind(this);
+        this.handleChange = this.handleChange.bind(this);
+    }
+
+    handleChange(e) {
```

Comment:
👍   Thanks for the suggestion here, cleared up a lot of unnecessary code and helped me get a better understanding of how `ref` functions!